components/file_attachment_list/file_attachment_list.jsx

```diff
@@ -91,17 +91,14 @@ export default class FileAttachmentList extends React.Component {
         }
 
         const postFiles = [];
-        let sortedFileInfos = [];
-
         if (fileInfos && fileInfos.length > 0) {
-            sortedFileInfos = fileInfos.sort((a, b) => a.create_at - b.create_at);
```

Comment:
No need to sort here since it's handled already by the redux store/utility (thru `makeGetFilesForPost` selector)